src/components/field.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+
+import styles from "./field.module.scss"
+import classNames from "classnames"
+
+
+interface Props {
+    className?: string
+    labelText?: string
+    labelFor?: string
+    contextualHelp?: string
+    error: string | undefined
```

Comment:
I think `error?: string` is the same thing and less verbose. Is there a reason you're explicitly making the type `string | undefined`?